syllabus/5-Substrate/1-Intro-to-Substrate_Slides.md

```diff
@@ -43,25 +38,23 @@ Highlight the multi-chain part.
 
 ### Why Substrate?
 
-<img rounded width="1000px" src="../../assets/img/5-Substrate/dev-4.1-maximalism.png" />
-
-Notes:
-
-Polkadot is the biggest bet in this ecosystem against chain maximalism, and Substrate plays a big
-role in this scenario.
-
----v
-
-### Why Substrate?
-
 - ⛓️ Future is multi-chain.
 <!-- .element: class="fragment" -->
 - 😭 Building a blockchain is hard. Upgrading it even harder.
 <!-- .element: class="fragment" -->
 - 💡 Framework!
 <!-- .element: class="fragment" -->
-- 🧐 But which attitude to take?
-<!-- .element: class="fragment" -->
+
+---v
+
+### Why Substrate?
+
+<img rounded width="1000px" src="../../assets/img/5-Substrate/dev-4.1-maximalism.png" />
+
+Notes:
+
+Polkadot is the biggest bet in this ecosystem against chain maximalism, and Substrate plays a big
```

Comment:
I think this is originally based on a  Gav quote, but I'm not sure it is really true anymore. If anything around here were a bet against maximalism it would be Substrate which actually allows you to build a sovereign chain if you want to. Cosmos SDK helps devs create sovereign chains only.

Reply:
This goes back to the intricate detail of what is sovereign and what is not. Polkadot assumes that its parachains are still sovereign, and by providing security to them, it is unblocking sovereign blockchains' main bottleneck: fragmentation of security. 

Needless to say, some part of me agrees with you. But I am inclined to keep this narrative as is, with an interpretation that is slightly in favor of Polkadot.

Reply:
Your call in the end. I will just point out that a parachain cannot even be live (let alone censorship resistant) without the help of relay chain validators. If the validators decide to censor a parachain, there is nothing the collators can do about it. This seems pretty clearly not sovereign to me.

In the abstract: If some chain B requires the cooperate of the authorities of another chain, A to include a block, then chain B is not sovereign.

I'm not trying to shit on the extremely valuable secure blockspace that polkadot is selling, and I think the para model is awesome. But awesome is not the same as sovereign.

Reply:
New version: 

> Substrate is a pure technological investment against chain maximalism, even outside of Polkadot. You
can create sovereign chains with substrate, or Polkadot Parachains, or more hybrid things.